### ISSUE TITLE
questions about training details

### ISSUE THREAD

**Issue by NeoForNew** [UkcheolShin/DRL-ISP#3]
Hi, I have several questions about the training details. 
1. Do I need to use the raw COCO dataset to train the detector? 
2. Is the Mask-RCNN trained using RAW images or RGB images? 
3. If I understand correctly. You directly use a trained Mask-RCNN model in the DRL training loop without training a Mask-RNN model yourself, right? 
Moreover, I want to add one segmentation function using your method. I have trained two DDRnet models(one uses Raw images and the other uses RGB images). Do you think I can directly use them in the DRL training loop?
Look forward to your reply. Your reply will be valuable to my research. Thank you!

**Comment by UkcheolShin** [comment]
Thanks for having an interest in our work.

1. No, you don't need to train the detection model on the raw dataset. But, you need the raw COCO dataset to train the RL model for the detection task. As described in the README, you can download the pre-processed dataset here (https://kaistackr-my.sharepoint.com/personal/shinwc159_kaist_ac_kr/_layouts/15/onedrive.aspx?id=%2Fpersonal%2Fshinwc159%5Fkaist%5Fac%5Fkr%2FDocuments%2FDRL%5FISP%2FDRL%5FISP%5FRAW&ga=1).
2. No, we used the pre-trained Mask-RCNN model provided in their original GitHub. We didn't fine-tune or train the model in the raw dataset. Our framework aims to generate desirable RGB images for the pre-trained model.
3. Yes, we use the pre-trained model only to estimate the reward for the object detection task. We didn't train the mask-RCNN model.
4. Yes, you can apply our framework to various tasks, including semantic segmentation. You only have to design a reward function for the segmentation task.
